### PR TITLE
feat: add reports to desk, create reports page

### PR DESCRIPTION
**Pull request by rvpasari**
https://app.asana.com/0/1198955099016653/1200351819266681

### CODE REVIEW

**Comment by rvpasari** [comment]
Screen Shot 2021-05-27 at 12 30 58 PM
Screen Shot 2021-05-27 at 12 30 42 PM

File: erpnext/config/desktop.py [hunk 1]
```diff
@@ -17,6 +17,15 @@ def get_data():
...
 		},
+		{
+			"module_name": "Reports",
+			"category": "Modules",
+			"label": _("Reports"),
+			"color": "#3498db",
+			"icon": "octicon octicon-repo",
+			"type": "module",
+			"description": "Accounts, billing, payments, cost center and budgeting."
+		},
```
Comment (hrwX) [inline erpnext/config/desktop.py]: - do not add the reports module before accounting.
- put it at the end of the existing modules

File: erpnext/config/reports.py [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+                {
+					"type": "report",
+                    "label": _("Employee CheckIn Report"),
+					"name": "Employee Checkin Report",
+					"doctype": "Employee Checkin",
+					"is_query_report": True
+				}
+			]
+		}
+    ]
```
Comment (hrwX) [inline erpnext/config/reports.py]: - Fix indentations.

**Review by hrwX — DISMISSED** [review]

File: erpnext/config/reports.py [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+			{
+				"type": "report",
+				"label": _("Employee CheckIn Report"),
+				"name": "Employee Checkin Report",
+				"doctype": "Employee Checkin",
+				"is_query_report": True
+			}
+		]
+		}
+	]
```
Comment (hrwX) [inline erpnext/config/reports.py]: - Indentation is still not correct, please fix it

**Review by hrwX — DISMISSED** [review]

**Review by hrwX — APPROVED** [review]

**Review by sahil28297 — APPROVED** [review]

**Pull request merged**